components/InteractionDetailsSection/InteractionDetailsSection.js

```diff
@@ -29,15 +29,14 @@ export default function InteractionDetailsSection({ interaction, contacts }) {
     });
 
   const participantsWithLinks = sortedActiveParticipants.map(
-    (sortedParticipant, index) => (
+    (sortedParticipant) => (
       <React.Fragment key={sortedParticipant.id}>
```

Comment:
Now the fragment is not needed anymore! Just pass the `key` to the `ParticipantLink` ^^